src/index.js

```diff
@@ -4,57 +4,93 @@ import storageAvailable from './Assets/Javascript/dataHandler.js';
 import './style.css';
 import dragIcon from './Assets/Images/dragicon.png';
 import refreshicon from './Assets/Images/refreshicon.png';
+import {
+  dragStart, dragEnd, dragOver, dragDrop,
+} from './Assets/Javascript/dragHandler.js';
 
 const toDoList = document.getElementById('to_do_list');
 const refreshIconImg = document.getElementById('refreshIcon');
 refreshIconImg.src = refreshicon;
 const newTaskInput = document.getElementById('newTaskInput');
 const taskList = new LinkedList();
-function tryalPopulate() {
-  taskList.add({ description: 'Clean my room', completed: false, index: null });
-  taskList.add({ description: 'Do my homework', completed: false, index: null });
-  taskList.add({ description: 'Take the trash out', completed: false, index: null });
-  taskList.indexify();
-  if (storageAvailable) {
-    localStorage.setItem('toDoList', JSON.stringify(taskList.head));
-  }
-}
-tryalPopulate();
-
-function build(linkList) {
+export default function build(linkList) {
   const removalItems = document.querySelectorAll('.taskItem');
   removalItems.forEach((elem) => {
-    elem.remove();
+    elem.remove(0);
   });
-  const objectsArray = [];
+  const objArray = [];
+
   const localInfo = JSON.parse(localStorage.getItem('toDoList'));
   linkList.head = localInfo;
-  const itemsArray = taskList.returnArray();
+  const itemsArray = taskList.returnFromIndex();
   for (let i = 0; i < itemsArray.length; i += 1) {
     const newTaskObj = document.createElement('li');
+    const newTaskObj2 = document.createElement('div');
     const checkBox = document.createElement('input');
-    const taskText = document.createElement('p');
+    const taskText = document.createElement('input');
+    const taskCompleted = document.createElement('input');
+
     const taskDragIcon = document.createElement('img');
-    objectsArray.push([toDoList, newTaskObj, 'taskItem', null, `taskItem_${i}`]);
-    objectsArray.push([newTaskObj, checkBox, 'taskCheckBox']);
-    objectsArray.push([newTaskObj, taskText, 'taskText', `${itemsArray[i].description}`]);
-    objectsArray.push([newTaskObj, taskDragIcon, 'dragIcon']);
+    objArray.push([toDoList, newTaskObj, 'taskItem', null, `taskItem_${i}`]);
+    objArray.push([newTaskObj, checkBox, 'taskCheckBox']);
+    objArray.push([newTaskObj, taskText, 'taskText']);
+    objArray.push([newTaskObj, taskCompleted, 'taskCompleted']);
+    objArray.push([newTaskObj, taskDragIcon, 'dragIcon']);
```

Comment:
[Optional]
On some browsers when you drag an image it automatically opens that image and makes the application harder to use. It will be better if you set this as a background image. 

![Screencast-from-07-17-2021-11_14_03-AM](https://user-images.githubusercontent.com/1642014/126030829-c4497b9b-e6e0-439c-968c-6424372e3f5b.gif)